models/user.js

```diff
@@ -97,6 +97,10 @@ const UserSchema = mongoose.Schema({
     type: Boolean,
     default: false,
   },
+  patron: {
```

Comment:
What's the purpose of this field? It doesn't appear to be set anywhere in the patreon routes.

Reply:
Created a PR:https://github.com/dekkerglen/CubeCobra/pull/2063